analyses/focal-cn-file-preparation/display-plots.md

```diff
@@ -0,0 +1,7 @@
+This markdown displays the plots produced in the `rna-expression-validation.R` script of this module. 
```

Comment:
I really like this little display markdown file. I'm going to have to use this idea in my analyses!